src/components/dataset/query/sidebar/panels/InfoViewDatasetAccess.jsx

```diff
@@ -0,0 +1,89 @@
+import React, { Fragment } from 'react';
+import PropTypes from 'prop-types';
+import { withStyles } from '@material-ui/core/styles';
+import { Grid, Link, Typography } from '@material-ui/core';
+import { connect } from 'react-redux';
+import DatasetAccess from '../../../DatasetAccess';
+
+const styles = (theme) => ({
+  root: {
+    display: 'block',
+    margin: theme.spacing(1),
+  },
+  learnMore: {
+    ...theme.mixins.jadeLink,
+    cursor: 'pointer',
+    textDecorationLine: 'underline',
+  },
+});
+
+const helpTitle = (
+  <Typography variant="h4">Roles and memberships in Terra Data Repository</Typography>
+);
+
+const helpContent = (
+  <Fragment>
+    <Typography variant="h6">Stewards</Typography>
+    <Typography>
+      Creating a dataset makes a user the Steward (or owner) of it. The steward of a dataset can:
+    </Typography>
+    <ul>
+      <li>Define the schema for a dataset</li>
```

Comment:
Hmm...the implies that the dataset's schema can be updated...

Reply:
This is pulled directly from our documentation. I'll remove it now, but we should update this when we do add schema migration